src/daemon/bitcoind/interface.rs

```diff
@@ -740,7 +740,7 @@ impl BitcoinD {
     pub fn get_wallet_transaction(
         &self,
         txid: &Txid,
-    ) -> Result<(String, Option<u32>, u32), BitcoindError> {
+    ) -> Result<(String, Option<u32>, Option<u32>, u32), BitcoindError> {
```

Comment:
Can you update the doc comment since you change the interface and apply the FIXME (a tuple with 3 values was already bad, with 4 this needs to be a struct.